.github/workflows/upstream_testing.yml

```diff
@@ -0,0 +1,13 @@
+---
+name: "Nautobot Upstream Monitor"
+
+on:  # yamllint disable-line rule:truthy rule:comments
+  schedule:
+    - cron: "0 4 */2 * *"  # every other day at midnight
```

Comment:
The comment says midnight but the crontab is 4am

Reply:
Fixed